common/config/presets/c_base_node.toml

```diff
@@ -38,13 +38,13 @@ identity_file = "config/base_node_id_nextnet.json"
 
 # Uncomment all gRPC server methods that should be denied default (only active when `grpc_enabled = true`)
 grpc_server_deny_methods = [
-    "get_version",
```

Comment:
Dont change these, this was a security finding that these should not be enabled by default

Reply:
We have a tracking issue for the base node to ask you about these thou: https://github.com/tari-project/tari/issues/5908

This should make it easier to enable in future

Reply:
Yeah, I forgot this here. Because I have to constantly comment them out, because I can't set them via command line.

Reply:
good point, we need to ensure its cli set-able as well.